changes/ce/fix-10746.en.md

```diff
@@ -0,0 +1 @@
+Add support for testing the `$events/delivery_dropped` event by the `rule_test` API.
```

Comment:
```suggestion
Add missing a missing event rule-engine event `$events/delivery_dropped`.
```

Reply:
The `$events/delivery_dropped` isn't missing, this schema only affects the `rule_test` API which is just for  testing an event in the dashboard